proto/v1/sql_service.proto

```diff
@@ -298,3 +302,56 @@ message StringifyMetadataRequest {
 message StringifyMetadataResponse {
   string schema = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
 }
+
+message ListQueryHistoryRequest {
+  // The name is the instance name to execute the query against.
+  // Format: instances/{instance}/databases/{databaseName}
```

Comment:
We can remove this name field now.

Reply:
Frontend updated here https://github.com/bytebase/bytebase/pull/11304/files#diff-042fc4ff4dc886954661d746451f14827a4c8f8fafeb05ac4e4dcc818bc10b0b